ansible/roles/sysprep/tasks/redhat.yml

```diff
@@ -60,7 +60,7 @@
     - name: Remove subscriptions
       rhsm_repository:
         name: '*'
-        state: absent
+        state: disabled
```

Comment:
Why is this change needed?

Reply:
If the user want to enable subscription at later stage, not sure if it will add the "removed" repos back. so I just opted to disable it instead of removing them.